tests/baselines/reference/findAllRefsMissingModulesOverlappingSpecifiers.baseline.jsonc

```diff
@@ -0,0 +1,43 @@
+// === findAllReferences ===
+// === /tests/cases/fourslash/findAllRefsMissingModulesOverlappingSpecifiers.ts ===
+// // https://github.com/microsoft/TypeScript/issues/5551
+// import { resolve/*FIND ALL REFS*/ as resolveUrl } from "idontcare";
```

Comment:
It's worth noting that this one just doesn't find any references while the other one finds the reference at the location of the request. I think it's fine - the most important thing is that both locations are not treated as references to the same thing.